src/dotnet/APIView/APIViewWeb/Managers/CommentsManager.cs

```diff
@@ -119,32 +136,71 @@ public async Task<CommentModel> UpdateCommentAsync(ClaimsPrincipal user, string
             return comment;
         }
 
-        public async Task DeleteCommentAsync(ClaimsPrincipal user, string reviewId, string commentId)
+        /// <summary>
+        /// Delete Comment
+        /// </summary>
+        /// <param name="user"></param>
+        /// <param name="reviewId"></param>
+        /// <returns></returns>
+        public async Task SoftDeleteCommentsAsync(ClaimsPrincipal user, string reviewId)
```

Comment:
What's the use case of this method to delete all comments?

Reply:
All comments for a review. Which should only happen when a review is deleted.